src/pages/editor/ChannelClaimTableRow.tsx

```diff
@@ -9,8 +11,13 @@ interface ChannelClaimTableRowProps {
   claimedChannel: ClaimedChannel;
 }
 const StyledTableCell = styled(TableCell)({
-  verticalAllign: 'top',
+  verticalAlign: 'top',
 });
+
+const StyledChip = styled(Chip)({
+  textWrap: 'nowrap',
```

Comment:
Antar dette ligger her for å unngå at innholdet kommer over to linjer, men ser ikke ut til at det har noen effekt?
![bilde](https://github.com/user-attachments/assets/e3d67384-4b45-42c5-b6c3-58123f947d86)

Er det noe ala dette du var ute etter?
```suggestion
    '.MuiChip-label': {
      whiteSpace: 'nowrap',
    },
```